package.json

```diff
@@ -81,6 +81,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
Why do you need this package here?

Reply:
I sincerely apologize if I overstepped. I was attempting to troubleshoot a bug that I believed was caused by my local environment. In hindsight, I realize that this package is not necessary for your application, and I apologize once again for any inconvenience caused and for my unintentional unprofessional behavior.

Reply:
@SourabhRajput123, everything looks good! Thanks for fixing that. 
Could you also please remove the changes from the `package-lock.json` file?